Hymns/HomeSearchView.swift

```diff
@@ -1,26 +1,19 @@
-//
-//  HomeSearchView.swift
-//  Hymns
-//
-//  Created by Benjamin Findeisen on 3/27/20.
-//  Copyright © 2020 skywalkerdude. All rights reserved.
-//
-
 import SwiftUI
 
 struct HomeSearchView: View {
```

Comment:
So in the Android app, the search view is different from the home view. i.e. the "search" bar in the home view is actually not editable. Rather, when you click it, it brings you to a new screen that does the search for you. That way the keyboard doesn't cover the navigation icons on the bottom

Reply:
I have attempted to address in commit c3070bd. It works and looks quite nice in my opinion. Let me know what you think and if it fully addresses the problem.